j5_zoloto/board.py

```diff
@@ -0,0 +1,58 @@
+"""Virtual Camera Board for detecting Fiducial Markers."""
+
+from pathlib import Path
+from typing import List, Optional, Set, Type, Union, cast
+
+from j5.backends import Backend
+from j5.boards import Board
+from j5.components import Component
+from zoloto.marker import BaseMarker
+
+from .component import MarkerCamera, MarkerCameraInterface
+
+
+class ZolotoCameraBoard(Board):
+    """Virtual Camera Board for detecting fiducial markers."""
+
+    name: str = "Zoloto Camera Board"
+
+    def __init__(self, serial: str, backend: Backend):
+        self._serial = serial
+        self._backend = backend
+
+        self._camera = MarkerCamera(0, cast(MarkerCameraInterface, backend))
+
+    @property
+    def serial_number(self) -> str:
+        """Get the serial number."""
+        return self._serial
+
+    @property
+    def firmware_version(self) -> Optional[str]:
+        """Get the firmware version of the board."""
+        return self._backend.firmware_version
+
+    def make_safe(self) -> None:
+        """Make this board safe."""
+        pass
```

Comment:
Worth closing the handle here?

```suggestion
        self._camera.close()
```

Reply:
f845e3177c897f4b4ab3ee1184a21aaf630bd3dc